vertexai/token-count/tokencount_test.go

```diff
@@ -26,19 +26,18 @@ import (
 func Test_countTokens(t *testing.T) {
 	tc := testutil.SystemTest(t)
 
-	prompt := "why is the sky blue?"
 	location := "us-central1"
 	modelName := "gemini-1.5-flash-001"
 
 	var buf bytes.Buffer
-	err := countTokens(&buf, prompt, tc.ProjectID, location, modelName)
+	err := countTokens(&buf, tc.ProjectID, location, modelName)
 	if err != nil {
 		t.Fatalf("Test_countTokens: %v", err.Error())
 	}
 
 	answer := buf.String()
 	s := strings.TrimPrefix(answer, "Number of tokens for the prompt: ")
-	s = strings.TrimSpace(s)
+	s = strings.Split(s, "\n")[0]
```

Comment:
Just curious about the change; are you trying to preserve any other leading or trailing whitespace other than specifically "\n"?

Reply:
There was a request to add additional print, therefore buf contains a bit longer string (desired output + new print). To properly adjust test I would need to split by "\n"

FWIW TrimSpace was redundant, cause previous line already removed space before the number 🙂